custom_components/tahoma/cover_devices/tahoma_cover.py

```diff
@@ -103,6 +103,9 @@ async def async_set_cover_tilt_position(self, **kwargs):
     def is_closed(self):
         """Return if the cover is closed."""
 
+        if self.current_cover_position is not None:
+            return self.current_cover_position == 0
```

Comment:
I will give priority to the position.

Reply:
Makes sense, with the amount of errors in the Overkiz API, this would be more reliable.